src/utils/prop-types.js

```diff
@@ -1,16 +1,20 @@
 import { shape, string, arrayOf, number, array } from 'prop-types';
 
+export const schema = shape({
+  /** Descriptive information about schema */
+  title: string,
+  description: string,
+  name: string,
+  /** Type of schema */
+  type: string,

```

Comment:
Is this something we should expose as an enum (i.e., oneOf([...]) rather than just string?

Reply:
That would be much better! 👍 Applied changes!